### PR TITLE
Contact form and finish the mobile version

### PR DESCRIPTION
**Pull request by Have-Samuel**
In Contact form and Mobile version Project we:

- Added a contact form 
- Applied the HTML & CSS Linters
- Documented work proffesionally
- Had to do it with peer-to-peer programming

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -632,3 +647,92 @@ ul a {
...
+  text-decoration: underline #87ceeb;
+  border: 2px solid #87ceeb;
+}
+
+.contact-section ul {
+  display: flex;
+  justify-content: center;
+  margin: 0 auto;
+  padding: 30px 0;
+}
```
Comment (julie-ify) [inline index.css:729]: I'm impressed by the hard work you put into this project, keep rocking 🔥. However, please let's stick to Figma designs and color. Consider using the correct color for these icons to stick to the design.

your design:
chrome_aVXfjmMQcm

Figma design:
chrome_0qEAspUDv5

File: index.css [hunk 2]
```diff
@@ -632,3 +647,92 @@ ul a {
...
+  width: 40%;
+  color: #fff;
+  margin: 24px auto;
+  margin-bottom: 136px;
+}
+
+.contact-button:hover {
+  text-decoration: underline #87ceeb;
+  border: 2px solid #87ceeb;
+}
```
Comment (julie-ify) [inline index.css]: - [ ] Please let's use the correct hover button interaction for the submit button so we can stick to Figma design.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi team, 

Good job so far!

## Highlight!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️  

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Whoistolu — APPROVED** [review]
### Hi team,
@Have-Samuel and @VicPeralta 

Your project is complete!  There is nothing else to say other than... It's time to merge.

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**